stable/jenkins/values.yaml

```diff
@@ -8,6 +8,10 @@
 # nameOverride:
 # fullnameOverride:
 
+# For FQDN resolving of the master service. Change this value to match your existing configuration.
+# ref: https://github.com/kubernetes/dns/blob/master/docs/specification.md
+# clusterZone: "cluster.local"
```

Comment:
you need to removed the comment otherwise we don't have a default value

Reply:
fixed.